libcxx/test/std/utilities/memory/pointer.traits/types.pass.cpp

```diff
@@ -0,0 +1,325 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Rename this test to `types.compile.pass.cpp`.

Reply:
Done.